data/styles/clear/include/Icons.mapcss

```diff
@@ -639,6 +640,7 @@ node|z16-[tourism=information][information=map]
 
 node|z16-[historic=boundary_stone],
 area|z16-[historic=boundary_stone],
+node|z16-[boundary=marker],
```

Comment:
mmm I don't think the monument icon is a good fit for boundary markers...
it'd be best to add a new icon e.g. based on `bcontrol-m`

for now it should be alright to use a `dot-m.svg` icon I think

Reply:
I moved it to `dot-m.svg` as you suggested.
I'm not sure about where to put it in file (to comply with ToC), so I chose 5.5 Amenity -> Others